src/patternfly/components/Dropdown/dropdown-group.hbs

```diff
@@ -0,0 +1,6 @@
+<{{#if dropdown-group--type}}{{dropdown-group--type}}{{else}}section{{/if}} class="pf-c-dropdown__group{{#if dropdown-group--modifier}} {{dropdown-group--modifier}}{{/if}}"
```

Comment:
can we also update the markup in https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Dropdown/dropdown-menu-groups.hbs to use the hbs partial?